builtin/credential/ldap/path_config_rotate_root_test.go

```diff
@@ -0,0 +1,64 @@
+package ldap
+
+import (
+	"context"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/vault/helper/testhelpers/ldap"
+	logicaltest "github.com/hashicorp/vault/helper/testhelpers/logical"
+	"github.com/hashicorp/vault/sdk/logical"
+)
+
+// This test relies on an external ldap server with a suitable person object (cn=admin,dc=planetexpress,dc=com)
+// with bindpassword "admin". - see the backend_test for more details.
+// This test will not run unless VAULT_ACC is set to something
+func TestRotateRoot(t *testing.T) {
+	if os.Getenv(logicaltest.TestEnvVar) == "" {
+		t.Skip("skipping rotate root tests because VAULT_ACC is unset")
+	}
```

Comment:
Is it possible that this could run in CI? I think I'm seeing other LDAP auth tests using `ldap.PrepareTestContainer()` that don't have this `VAULT_ACC` guard .

Reply:
They end up having it implicitly, since `logicaltest.Test` includes the `VAULT_ACC` guard.

Reply:
Ahh, gotcha! Makes sense thanks 👍